qcodes/instrument_drivers/basel/sp983c.py

```diff
@@ -38,6 +39,12 @@ def __init__(self, name: str, **kwargs: Any):
                            vals=Enum(30., 100., 300., 1e3, 3e3, 10e3, 30e3,
                                      100e3, 1e6))
 
+        self.add_parameter('bias_volt',
+                           initial_value=0,
+                           unit='mV',
```

Comment:
The source instruments to supply the bias voltage will be with units volts so think we should change this as unit V and change the vals to match.